source/blog/2020-04-06-release-notes.md

```diff
@@ -0,0 +1,79 @@
+---
+layout: post
+title: April 2020 Release Notes - Version 20.04.0
+published: true
+post_author:
+  display_name: Ian Bennett
+  twitter: enbnt
+tags: Releases, Finagle, Finatra, Scrooge, TwitterServer, Util
+---
+
+In these unprecedented times, we wish for nothing but your health and safety. April showers 🌦️ bring May flowers 🌻, so #StayAtHome and fill some time reading about some of the changes in our latest release.
+
+### [Finagle](https://github.com/twitter/finagle/) ###
```

Comment:
I don't think this should have the trailing ###?

Reply:
I looked back at our prior releases and this is the formatting that has been used consistently. I don't think it's needed, but it's consistent.